utils/python/CIME/XML/env_batch.py

```diff
@@ -93,16 +97,17 @@ def get_values(self, item, attribute={}, resolved=True, subgroup=None):
                 # seach in all entry nodes
                 for node in nodes:
 
-                    # Init and construct attribute path
-                    attr      = None
+                    
+                    # Build return structure
+                    attr      = node.get('id')
+                    g         = None
```

Comment:
I'm seeing some variable names that are pretty undescriptive: "g", "r", "t". Could we maybe expand these to full words?